### PR TITLE
fix: argo workflows failed to install

### PR DESCRIPTION
**Pull request by alexcodelf**
**Problem:**

Walrus crashed and restarted because argo workflows failed to install.

**Solution:**

Add permission for apply walrus in k8s, using `rest.Config` directly instead of converting to`clientcmdapi.Config`

**Related Issue:**

#1644

### CODE REVIEW

File: pkg/k8s/deploy/helm.go [hunk 1]
```diff
@@ -4,13 +4,18 @@ import (
 	"fmt"
 	"io"
 	"os"
-	"path"
 	"time"
 
+	"k8s.io/client-go/restmapper"
+
```
Comment (gitlawr) [inline pkg/k8s/deploy/helm.go:10]: remove this line

File: pkg/workflow/client.go [hunk 2]
```diff
@@ -406,8 +406,8 @@ type ArgoAPIClient struct {
 }
 
 func NewArgoAPIClient(restCfg *rest.Config) *ArgoAPIClient {
-	apiConfig := k8s.ToClientCmdApiConfig(restCfg)
-	clientConfig := clientcmd.NewDefaultClientConfig(apiConfig, nil)
+	apiConfig, _ := k8s.ToClientCmdApiConfig(restCfg)
```
Comment (gitlawr) [inline pkg/workflow/client.go:409]: Why do we ignore the error?

Comment (alexcodelf) [reply pkg/workflow/client.go]: The reason is we follow the pattern of init API handler not return error. Let expose it when init API server.

**Review by gitlawr — DISMISSED** [review]
LGTM

**Pull request merged**